pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -725,9 +725,13 @@ def qiskit_to_tk(qcirc: QuantumCircuit, preserve_param_uuid: bool = False) -> Ci
     """
     Converts a qiskit :py:class:`qiskit.QuantumCircuit` to a pytket :py:class:`Circuit`.
 
+    *Note:* Support for conversion of symbolic circuits is currently limited. In
```

Comment:
From my understanding this is still missing that point that the `Parameterexpression.simpify()` might fail for the parameters in the constructed qiskit circuit.

Reply:
Do you have some example code showing this issue?

Reply:
I meant code using pytket-qiskit functions that doesn't work as expected...? Users aren't expected to use `ParameterExpression` directly.

Reply:
Yes please, then we can add it as a test (which we'll mark as `xfail`).

Reply:
testcase is added, feel free to merge it now if you are happy with the test